.github/workflows/container-image.yml

```diff
@@ -64,6 +87,36 @@ jobs:
       with:
         image: "samba-ad-server:latest"
         retention_days: 1
+  build-client-centos:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - name: Build the centos client image
+      run: make OS_NAME=centos build-client
+    # Here we upload samba-client image to artifacts locally for consumption
+    # during samba-toolbox build process.
+    - name: Upload client image
+      uses: ishworkh/docker-image-artifact-upload@v1
+      with:
+        image: "quay.io/samba.org/samba-client:centos-latest"
+        retention_days: 1
+
+  build-toolbox-centos:
+    needs: build-client-centos
+    runs-on: ubuntu-latest
+    env:
+      BUILDAH_FORMAT: oci
+    steps:
+    - uses: actions/checkout@v3
+    # Download locally stored samba-client image to be used as base for building
+    # samba-toolbox.
+    - name: Download client image
+      uses: ishworkh/docker-image-artifact-download@v1
+      with:
+        image: "quay.io/samba.org/samba-client:centos-latest"
+    - name: Build the toolbox image
+      run: make OS_NAME=centos build-toolbox
```

Comment:
```
make: *** No rule to make target 'images/toolbox/Containerfile.centos', needed by '.build.centos-toolbox'.
```

Reply:
_images/toolbox/Containerfile.centos_ was missing for which I've raised https://github.com/samba-in-kubernetes/samba-container/pull/126.

Reply:
@anoopcs9  wrote:

> ```
> make: *** No rule to make target 'images/toolbox/Containerfile.centos', needed by '.build.centos-toolbox'.
> ```

right. somehow this file has vanished.  dom't know how ...  

I am working on resurrecting it ...

Reply:
#126 is now merged with required Containerfile and `build-toolbox-centos` passed.